packages/salesforcedx-vscode-apex/src/commands/forceApexTestRunCodeAction.ts

```diff
@@ -4,12 +4,15 @@
  * Licensed under the BSD 3-Clause license.
  * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
-
+import { HumanReporter, TestItem, TestService } from '@salesforce/apex-node';
+import { TestLevel } from '@salesforce/apex-node/lib/src/tests/types';
```

Comment:
Let's make sure we surface this type at the top of the library so we can just import it by using `@salesforce/apex-node`.